modules/oadp-release-notes-1-1-1.adoc

```diff
@@ -0,0 +1,23 @@
+
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-1_{context}"]
+= OADP 1.1.1 release notes
+
+[id="known-issues-1-1-1_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+* OADP currently does not support backup and restore of AWS EFS volumes using restic in Velero (link:https://issues.redhat.com/browse/OADP-778[*OADP-778*]).
+
+* CSI backups might fail due to a Ceph limitation of 100 `VolumeSnapshotContent` snapshots per PVC
```

Comment:
Because this is detailed further below, and the number depends on CephFS or RBD, is it possible to group these together and exclude the number on this particular line?

Reply:
Done.